.github/workflows/cockpit-lib-update.yml

```diff
@@ -0,0 +1,32 @@
+name: cockpit-lib-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
+  # can be run manually on https://github.com/cockpit-project/starter-kit/actions
+  workflow_dispatch:
+jobs:
+  cockpit-lib-update:
+    permissions:
+      contents: write
+      pull-requests: write
+      statuses: write
```

Comment:
Does this need statuses? I don't see anything setting a status... If it's needed for the github api in script from bots, please add a comment so it makes sense at a glance.

Reply:
Ah, it probably doesn't. So we can drop that copy-pasta.